R/github.R

```diff
@@ -225,8 +225,15 @@ use_github_links <- function(auth_token = deprecated(),
   res <- gh("GET /repos/{owner}/{repo}")
 
   desc <- proj_desc()
-  if (!res$html_url %in% desc$get_urls()) {
-    use_description_field("URL", res$html_url, overwrite = overwrite)
+  existing_urls <- desc$get_urls()
+  
+  if (!res$html_url %in% existing_urls) {
+    if (overwrite || length(existing_urls) == 0) {
+      urls <- res$html_url
+    } else {
+      urls <- paste(existing_urls, res$html_url, sep = ", ", collapse = ", ")
+    }
+    use_description_field("URL", urls, overwrite = TRUE)
```

Comment:
This kind of short-circuits the overwrite argument but has the intended effect, without rewriting `use_description_field()`